vendor/modules.txt

```diff
@@ -600,3 +600,4 @@ sigs.k8s.io/structured-merge-diff/v4/value
 ## explicit; go 1.12
 sigs.k8s.io/yaml
 sigs.k8s.io/yaml/goyaml.v2
```

Comment:
do we need both? Or this is transitive?

Reply:
I believe this is just transitive, it was introduced when bumping from 1.3.0 to 1.4.0 which as well has goyaml.v3.

Reply:
so we need both in vendor folder? What uses it?

Reply:
Seems that way.

https://github.com/kubernetes-sigs/yaml/blob/b9a9b1cb9d17f91e923529e5dcd3df1a6500b6a3/yaml.go#L27

The primary module references and utilizes the `sigs.k8s.io/yaml/goyaml.v2` and `sigs.k8s.io/yaml/goyaml.v3`.

Reply:
/lgtm